src/report.js

```diff
@@ -96,11 +96,11 @@ _WV.$functions[F_SHOWREPORT] = function (reportFormat) {
 	///////////////////////////////////////////////////////////////////////
 	// Support functions
 	function getLayerParam() {
-		var mask;
+		/*var mask;

```

Comment:
Let's completely remove the getLayerParam() function then?

Reply:
Yes, the next PR will have it removed and some other tweaks. Have a few people testing it to help find where things are broken.